### DISCUSSION TITLE
Welcome to pygame-markdown Discussions!

### DISCUSSION THREAD

**Discussion by CribberSix** [CribberSix/pygame-markdown#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded.

**Comment by larstuff** [comment]
Hi. Interesting project. How do I install this package? 
I'm a newbie at github, so excuse me if the question seems trivial.

**Reply by CribberSix** [reply]
Hi @larstuff , the package is published on [pypi](https://pypi.org/project/pygame-markdown/) so you can install it as any other package with  `pip install pygame-markdown`!
